packages/extension/src/components/icon/keystone-text.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import { IconProps } from "./types";
+
+export const KeystoneTextIcon: FunctionComponent<IconProps> = ({
```

Comment:
@renfengshi Could you look at another icon file and follow the convention for icon files?